src/test/java/org/gridsuite/actions/test/GenerateScriptFromFiltersTest.java

```diff
@@ -27,26 +27,22 @@ public void generateScriptTest() {
         countries.add("FR");
         countries.add("BE");
 
-        assertEquals("import org.gridsuite.actions.server.utils.FiltersUtils;\n" +
-            "\n" +
-            "for (equipment in network.generators) {\n" +
-            "  if ((equipment.terminal.voltageLevel.nominalV == 90.0)\n" +
-            "      && (FiltersUtils.matchID('BRESS*', equipment) || FiltersUtils.matchName('OTHER*', equipment))\n" +
-            "      && FiltersUtils.isLocatedIn(['FR','BE'], equipment)\n" +
-            "     ) {\n" +
-            "        contingency(equipment.id) { equipments equipment.id }\n" +
-            "  }\n" +
-            "}\n", filtersToScript.generateGroovyScriptFromFilters(new FiltersContingencyListAttributes(null, null, "BRESS*",
+        assertEquals("for (equipment in network.generators) {\n" +
```

Comment:
In all theses example, you don't need to specify the FiltersUtils class? FiltersUtils.matchID('BRESS*', equipment) => matchID('BRESS*', equipment)

Reply:
Done